build-system/tasks/e2e/describes-e2e.js

```diff
@@ -462,23 +428,12 @@ function describeEnv(factory) {
      * @return {Set<string>}
      */
     function getAllowedBrowsers() {
-      const {engine, browsers} = getConfig();
+      const {browsers} = getConfig();
```

Comment:
Can you remove the `engine` flag in the e2e task as well?

Reply:
Great catch.